app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[0] != "cp":
+        raise ValueError("You line should start with cp")
+    if len(command) != 3:
+        raise ValueError("Write down cp than file name "
+                         "where the info is and than "
+                         "file name where you w ant to safe that info")
+    if command[1] == command[2]:
+        raise ValueError("There should be two different names of files")
+    elif ".txt" in (command[1] and command[2]):
```

Comment:
In this line, you're checking if '.txt' is in both `command[1]` and `command[2]`. However, the way it's written currently, it only checks if '.txt' is in `command[1]`. To check both, you should write: `elif '.txt' in command[1] and '.txt' in command[2]`.